state/remote/state.go

```diff
@@ -154,6 +173,7 @@ func (s *State) PersistState() error {
 		return err
 	}
 
+	log.Print("[TRACE] state/remote: persisting new state via client")
```

Comment:
This applies to all remote states -- do you believe this log line adds value? Inquiring as our logs have a lot of information as-is.

Reply:
Yea - I think maybe we'll just remove all the traces I added before merging this PR? I'd guess it's of marginal value since the `Put` will error with a hopefully useful message with the other providers.